students/sikorski_marcin/lesson_9&10/fillingInTheGaps.py

```diff
@@ -0,0 +1,16 @@
+import shutil, os, re
+
+sequenceToUse = re.compile(r'^(spam)(\d)(.txt)')
+ext = ".txt"
+
+
+def nameLocator(location):
+    listOfFiles = []
+    for root, dirs, files in os.walk(location):
+        if sequenceToUse in files:
+            listOfFiles.append(os.path.join(root, sequenceToUse))
+        return listOfFiles
+    print(listOfFiles)
```

Comment:
to jest nie skończone zadanie, ponieważ w podstawowej wersji program powinien znaleźć dziury w numeracji i następnie zmienić nazwy plików (przenumerować) tak, żeby usunąć dziury.
Czyli jeżeli mamy pliki:

- spam_001.txt
- spam_004.txt
- spam_010.txt
 to wynikiem działania skryptu powinien być zestaw plików:

- spam_001.txt
- spam_002.txt
- spam_003.txt

Druga wersja programu (trudniejsza, opcjonalna) to dodając nowy plik, program uzupełnia dziury:
- spam_001.txt
- spam_004.txt
- spam_010.txt
dodajemy plik:
- spam_001.txt
- spam_002.txt # nowy plik
- spam_004.txt
- spam_010.txt